extractor/filesystem/language/php/composerlock/composerlock_test.go

```diff
@@ -198,6 +198,50 @@ func TestExtractor_Extract(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "drupal_packages",
+			InputConfig: extracttest.ScanInputMockConfig{
+				Path: "testdata/drupal-packages.json",
+			},
+			WantPackages: []*extractor.Package{
+				{
+					Name:      "drupal/core",
+					Version:   "10.4.5",
+					PURLType:  purl.TypeComposer,
+					Locations: []string{"testdata/drupal-packages.json"},
+					SourceCode: &extractor.SourceCodeIdentifier{
+						Commit: "5247dbaa65b42b601058555f4a8b2bd541f5611f",
+					},
+					Metadata: osv.DepGroupMetadata{
+						DepGroupVals: []string{},
+					},
+				},
+				{
+					Name:      "drupal/tfa",
+					Version:   "2.0.0-alpha4",
+					PURLType:  purl.TypeDrupal,
+					Locations: []string{"testdata/drupal-packages.json"},
+					SourceCode: &extractor.SourceCodeIdentifier{
+						Commit: "2.0.0-alpha4",
```

Comment:
I think this is separate bug whereby we're assuming `dist.reference` is always a SHA when actually it's not - I can do another PR fixing this

Reply:
I've opened https://github.com/google/osv-scalibr/pull/958 for this